### PR TITLE
bin

### PR DESCRIPTION
**Pull request by Nidhu418**
please review

### CODE REVIEW

File: comcastCRM/src/test/java/comcastCRM/P2.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package comcastCRM;
+
+import org.testng.annotations.Test;
+
+public class P2 {
+	@Test
+	public void add()
+	{
+		System.out.println("Hiiiiiiiii  iiiiiiiiii");
+	}
```
Comment (Nidhushree) [inline comcastCRM/src/test/java/comcastCRM/P2.java:10]: rewived

**Review by Nidhushree — COMMENTED** [review]
reviwed done

**Comment by Nidhu418** [comment]
fixed

**Comment by Nidhu418** [comment]
fixed123

File: comcastCRM/src/test/java/comcastCRM/P2.java (hunk 1 above)
Comment (Nidhu418) [reply comcastCRM/src/test/java/comcastCRM/P2.java:10]: fixedddddd

**Comment by Nidhushree** [comment]
merged

**Pull request merged**